README.md

```diff
@@ -58,29 +59,85 @@ Build the project by navigating to the `iOS` folder on your terminal, and typing
 carthage update --platform iOS
 ```
 
-Create a folder called `Frameworks` in the `iOS` folder. Find the `Lumina.framework` and `VisualRecognitionV3.framework` from the `Carthage Build` folder, and move them to the `Frameworks` folder.
-
-Open the `rainbow.xcodeproj` in XCode. Select the rainbow target, choose the General tab at the top, and scroll down to the Linked Frameworks and Libraries section at the bottom.
-
-Drag and drop the `Frameworks` folder to the rainbow project - click `add groups` instead of add folder references. Navigate to the `embedded binaries` section of the rainbow project general information, and add the two frameworks as embedded binaries.
-
-### To develop the server
-
-1. Provision a Cloudant NoSQL DB instance
-2. Provision a Push Notification instance
-3. Deploy the app
-
-More details to come!
-
-# Build your own version!
-
-1. Pick a theme and set of items -- museum pieces, office hardware, conference booths, whatever
-2. Create a model in Watson Studio -- details to come
-3. Replace the model at [iOS/rainbow/Model/ProjectRainbowModel_1753554316.mlmodel](iOS/rainbow/Model/ProjectRainbowModel_1753554316.mlmodel) 
-4. Update the JSON file that lists the objects [iOS/rainbow/Config/GameObjects.json](iOS/rainbow/Config/GameObjects.json)
-5. If you need icons check out [https://thenounproject.com/](https://thenounproject.com/)
-
-More details to come!
+### Setting Up Your [Kitura](http://kitura.io) Server
+
+1. Go to the [IBM Cloud console](https://console.bluemix.net), and click Create Resource.
+2.  Search for "Cloudant NoSQL DB" and create a service. Take note of the name of the created service. 
+<p align="center">
+	<img src="./images/cloudantChoice.png" width=300>
+</p>
+3. Go to your Cloudant service home page, and click the green  **Launch** button. Click the database icon on the left, and along the top, click "Create Database". Name it `routes-users`, and click "Create Document". Edit your JSON to include a `"username"` and a `"password"` of your choosing. If you do this, you will need this when you set up your iOS application.
+4. Do the same thing for a Push Notifications service as well.
+ <p align="center">
+	<img src="./images/pushNotificationsChoice.png" width=300>
+</p>
+ 
+ (**to set up push notifications with your app, you will need to follow the [guide](https://console.bluemix.net/docs/services/mobilepush/index.html#gettingstartedtemplate) for embedding them into your app**)
```

Comment:
use a `>` for a note instead

Reply:
Ok, let's add that after because I am struggling to see how that affects it